bundles/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/internal/handler/ExecHandler.java

```diff
@@ -299,3 +354,41 @@ public void run() {
     }
 
 }
+
+
+/**
+ * Contains information about which operating system openHAB is running on.
+ * Found on https://stackoverflow.com/a/31547504/7508309, slightly modified
+ *
+ * @author Constantin Piber (for Memin) - Initial contribution
+ */
+class Util {
```

Comment:
I'd recommend making these methods part of the ExecHandler class directly. If not, this class needs to be in a new file.